ddl/ddl_worker.go

```diff
@@ -151,7 +151,14 @@ func (w *worker) typeStr() string {
 }
 
 func (w *worker) String() string {
-	return fmt.Sprintf("worker %d, tp %s", w.id, w.typeStr())
+	tp := "unknown"
```

Comment:
It looks strange. I know if we change the `typeStr` func will cause some compatibility problems.

Reply:
At least it is more informative to users than "tp add index".